test/integration/test_operations.rb

```diff
@@ -56,18 +56,52 @@
 
         val1 = 'meta'
         dc.set('meta_key', val1)
-        val2, meta_flags = dc.get('meta_key', meta_flags: %i[l h t])
+        val2, meta_flags = dc.get('meta_key', meta_flags: %i[l h t c])
 
         assert_equal val1, val2
         # not yet hit, and last accessed 0 from set
-        assert_equal({ c: 0, l: 0, h: false, t: -1, bitflag: nil }.sort, meta_flags.sort)
+        assert meta_flags.delete(:c)
+        assert_equal({ l: 0, h: false, t: -1, bitflag: nil }.sort, meta_flags.sort)
```

Comment:
at the moment bitflags is always present as it is a bit outside of a legacy flag... we could try to refactor it a away it is a bit awkward as a legacy flag, but there is meta flag option `-f` to request it so we could get rid of it, some of the dalli methods expect it so it might require a bit larger refactoring.